version/string_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package version
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestVersionsGetString(t *testing.T) {
```

Comment:
A bit surprised this isn't done with a table test... But with only two cases... I leave it up to you on if you feel like it's worthwhile.